erpnext/healthcare/doctype/patient_appointment/test_patient_appointment.py

```diff
@@ -21,6 +21,8 @@ def test_status(self):
 		frappe.db.set_value('Healthcare Settings', None, 'automate_appointment_invoicing', 0)
 		appointment = create_appointment(patient, practitioner, nowdate())
 		self.assertEquals(appointment.status, 'Open')
+		create_therapy_session(appointment)
+		self.assertEquals(frappe.db.get_value('Patient Appointment', appointment.name, 'status'), 'Closed')
```

Comment:
Similarly, check whether status is getting changed to Open on therapy session cancellation. Test the same for Patient Encounter.